### PR TITLE
zipkin tracing

### PR DESCRIPTION
**Pull request by Somesh**
Initialise zipkin tracing

### CODE REVIEW

File: tracer/appdash.go [hunk 1]
```diff
@@ -12,12 +12,10 @@ import (
 	"sourcegraph.com/sourcegraph/appdash"
 	appdashot "sourcegraph.com/sourcegraph/appdash/opentracing"
 	"sourcegraph.com/sourcegraph/appdash/traceapp"
-
-	"gopkg.in/tokopedia/logging.v1"
 )
 
 func InitAppdash(cfg *Config) {
-	logging.Debug.Println("starting tracer on ", cfg.Port)
```
Comment (bmishra) [inline tracer/appdash.go]: Keep them as debug?

File: tracer/zipkin.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		)
+		logText = logText + " Kafka"
+	}
+
+	if err != nil {
+		fmt.Println("err", "unable to create collector", "fatal", err)
+		return
+	}
+
+	fmt.Println(logText)
```
Comment (bmishra) [inline tracer/zipkin.go]: Can remove or log as debug info

File: tracer/zipkin.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	} else {
+		collector, err = zipkin.NewKafkaCollector(
+			strings.Split("127.0.0.1:9092", ","), // default kafka port
+			zipkin.KafkaLogger(log.NewNopLogger()),
+		)
+		logText = logText + " Kafka"
+	}
+
+	if err != nil {
+		fmt.Println("err", "unable to create collector", "fatal", err)
```
Comment (bmishra) [inline tracer/zipkin.go]: Use log.Println or move to debug

**Review by bmishra — CHANGES_REQUESTED** [review]

**Review by bmishra — APPROVED** [review]

**Review by qzaidi — COMMENTED** [review]
we can use go plugins here - to make this pluggable for any new tracer :)

**Pull request merged**